src/lib/eslint/JavascriptEslintStrategy.ts

```diff
@@ -7,17 +7,23 @@ import path = require('path');
 import {ESLint} from 'eslint';
 
 const ES_CONFIG: ESLint.Options = {
-	"baseConfig": {},
 	"overrideConfig": {
+		"parser": "@babel/eslint-parser",
 		"parserOptions": {
-			"sourceType": "module",
-			"ecmaVersion": 2018,
+			"requireConfigFile": false,
+			"babelOptions": {
+				"parserOpts": {
+					"plugins": ["classProperties", ["decorators", {"decoratorsBeforeExport": false}]]
+				}
+			}
 		},
 		"ignorePatterns": [
 			"node_modules/!**"
 		]
 	},
-	"useEslintrc": false // Will not use an external config
+	"useEslintrc": false, // Will not use an external config
+	"resolvePluginsRelativeTo": __dirname, // Use the plugins found in the sfdx scanner installation directory
```

Comment:
I wonder how this would work in the packaged format of sfdx-scanner. If we need to include something in our package, we may have to make changes to package.json.

Reply:
I'm not entirely sure why/if that line is needed, I was following the format of LWCEslintStrategy, but it seems to work the same either way.

Reply:
Looks like the dependencies have already been added to our package.json.